docs/mp/health/01_introduction.adoc

```diff
@@ -114,24 +138,369 @@ microservice reports itself as ready.
 |Treats the instance as still starting up; does not check liveness or readiness until the startup probe reports success or times out according to its configuration.
 |===
 
-=== Built-in and Custom Health Checks
 
-==== Built-in Health Checks
-Helidon provides built-in, default checks for each endpoint.
-The built-in liveness checks include various environmental values, such as whether the JVM has detected deadlocks
-or whether there is sufficient heap space. The built-in readiness and startup checks always report `UP`.
+== Configuration
+
+Health checks may be configured using the following properties.
+
+The responsible for configuration class is:
+
+include::../../shared/config/io.helidon.health.HealthSupport.adoc[]
+
+Current properties may be set in `application.yaml` or in `microprofile-config.properties` with `health` prefix.
+
+For example, you can specify a custom port and root context for the root health endpoint path.
+However, you cannot use different ports, such as http://localhost:8080/myhealth and http://localhost:8081/myhealth/live.
+Likewise, you cannot use different paths, such as http://localhost:8080/health and http://localhost:8080/probe/live.
+The example below will change the root path.
+[source,yaml]
+.Create a file named `application.yaml` in the `resources` directory with the following contents:
+----
+health:
+  web-context: "myhealth"  // <1>
+----
+<1> The web-context specifies a new root path for the health endpoint.
+
+
+== Reference
+
+TODO:
```

Comment:
I think a TODO does not belong here, if this should be our reference document

Reply:
`Reference` moved to the end.